scripts/install

```diff
@@ -105,7 +105,7 @@ setfacl -R -m g:"www-data":rwX -m u:$app:rwX "$final_path/var/"
 ynh_script_progression --message="Configuring PHP-FPM..." --weight=1
 
 # Create a dedicated PHP-FPM config
-ynh_add_fpm_config
+ynh_add_fpm_config --usage=low --footprint=low
```

Comment:
```suggestion
ynh_add_fpm_config
```

cf recent changes in ynh_add_fpm_config, we don't need to specify `--usage=low --footprint=low` as long as the extra_php-fpm.conf file exists